node/malus/src/variants/suggest_garbage_candidate.rs

```diff
@@ -37,6 +38,8 @@ use polkadot_primitives::v2::{CandidateDescriptor, CandidateHash};
 use polkadot_node_subsystem_util::request_validators;
 use sp_core::traits::SpawnNamed;
 
+use rand::distributions::{Bernoulli, Distribution};
```

Comment:
Is it worth to support multiple distributions ?

Reply:
In this case the `.sample()` function comes from `Distribution` and not `Bernoulli` so it is necessary.